apps/publish_mdm/tables.py

```diff
@@ -57,3 +58,38 @@ class Meta:
         template_name = "patterns/tables/table.html"
         attrs = {"th": {"scope": "col", "class": "px-4 py-3 whitespace-nowrap"}}
         orderable = False
+
+
+class DeviceTable(tables.Table):
+    """A table for listing MDM Devices."""
+
+    last_seen_mdm = tables.DateTimeColumn(
+        accessor="latest_snapshot__last_sync", verbose_name="Last seen (MDM)"
+    )
+    last_seen_vpn = tables.DateTimeColumn(verbose_name="Last seen (VPN)")
+
+    class Meta:
+        model = Device
+        fields = [
+            "serial_number",
```

Comment:
My work profile device has no serial number; perhaps we could include the `device_id` for now, too.
```suggestion
            "device_id",
            "serial_number",
```